appendices/editor-checks-template.md

```diff
@@ -0,0 +1,21 @@
+```markdown
```

Comment:
to avoid confusion maybe we should change the name of this file to editor-initial-response-template.md